modules/compute/container-registry/.bicep/nested_privateEndpoint.bicep

```diff
@@ -0,0 +1,58 @@
+param location string
```

Comment:
Let's make a note that we will need to update this later in another PR to consume the private endpoint bicep module from the network/private-endpoint module instead to avoid duplication. 

![image](https://github.com/DEFRA/bicep-modules/assets/79708209/537782bc-467c-47a8-a606-8ec8eb53deb7)